### PR TITLE
CSP-408 - Namespaces and references updated to .AANHub.

### PR DESCRIPTION
**Pull request by LiamWright**
References of `*.AAN.*` or `*.AAN.Hub.*` to `*.AANHub.*`

Also, updated certain code standards to current .NET recommendations. (E.G: Prefer var, etc)

**Local changes needed!**

Update LOCAL entry in config table:
`RowKey: SFA.DAS.AANHub.Api_1.0`
image

### CODE REVIEW

**Review by ianatesfa — APPROVED** [review]

**Review by HussainsGitHub — APPROVED** [review]

File: src/SFA.DAS.AANHub.Application/Commands/CreateMember/CreateMemberCommandHandler.cs [hunk 1]
```diff
@@ -32,31 +29,31 @@ public class CreateMemberCommandHandler : IRequestHandler<CreateMemberCommand, C
...
-                    Deleted = null,
-                    Status = MembershipStatuses.Live.ToString()
-                }
-            );
-            await _membersContext.SaveChangesAsync();
+            await _membersContext.Entities.AddAsync(member, cancellationToken);
+            await _membersContext.SaveChangesAsync(cancellationToken);
 
-            long id = long.TryParse(command.id, out id) ? id : 0;
+            long id = long.TryParse(command.Id, out id) ? id : 0;
```
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Application/Commands/CreateMember/CreateMemberCommandHandler.cs:51]: Can be simplified to `long.TryParse(command.Id, out var id);`

File: src/SFA.DAS.AANHub.Application/Commands/CreateMember/CreateMemberCommandHandler.cs [hunk 2]
```diff
@@ -32,31 +29,31 @@ public class CreateMemberCommandHandler : IRequestHandler<CreateMemberCommand, C
 
         public async Task<CreateMemberResponse> Handle(CreateMemberCommand command, CancellationToken cancellationToken)
         {
-            Guid memberId = Guid.NewGuid();
+            var memberId = Guid.NewGuid();
+
```
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Application/Commands/CreateMember/CreateMemberCommandHandler.cs:33]: This handler is breaking single responsibility principle

File: src/SFA.DAS.AANHub.Application/Commands/CreateMember/CreateMemberResponse.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Commands.CreateMember
+{
+    public class CreateMemberResponse
+    {
+        public Member? Member { get; set; }
```
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Application/Commands/CreateMember/CreateMemberResponse.cs:8]: I don't see why this has to be nullable, the handler always sets a value.

File: src/SFA.DAS.AANHub.Domain/Entities/Member.cs [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-namespace SFA.DAS.AAN.Domain.Entities
+namespace SFA.DAS.AANHub.Domain.Entities
 {
     public class Member : EntityBase
     {
         public Guid Id { get; set; }
-        public string UserType { get; set; }
-        public string Status { get; set; }
+        public string? UserType { get; set; }
+        public string? Status { get; set; }
```
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Domain/Entities/Member.cs:8]: No navigation properties?

File: src/SFA.DAS.AANHub.Domain/Enums/MembershipStatuses.cs [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ﻿
-namespace SFA.DAS.AAN.Domain.Enums
+namespace SFA.DAS.AANHub.Domain.Enums
 {
     public enum MembershipStatuses
```
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Domain/Enums/MembershipStatuses.cs:4]: Enum should have singular name `MembershipStatus`

File: src/SFA.DAS.AANHub.Domain/Interfaces/IDateTimeHelper.cs [hunk 6]
```diff
@@ -1,15 +1,18 @@
...
     }
-
+    [ExcludeFromCodeCoverage]
     public class UtcTimeProvider : IDateTimeHelper
     {
         public DateTime Now => DateTime.UtcNow;
     }
-
+    [ExcludeFromCodeCoverage]
     public class SpecifiedTimeProvider : IDateTimeHelper
```
Comment (DevenKShah) [inline src/SFA.DAS.AANHub.Domain/Interfaces/IDateTimeHelper.cs:16]: Where is this being used?

**Review by DevenKShah — COMMENTED** [review]
There are few comments I have added (and few I have not). I appreciate most of them are not directly related to this namespace change so I don't expect them to be fixed as part of this PR, but we do need to address few things, we can discuss this and create tasks accordingly.

**Pull request merged**